src/main/java/org/broadinstitute/hellbender/utils/variant/VariantContextVariantAdapter.java

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.hellbender.utils.variant;
+
+import htsjdk.variant.variantcontext.VariantContext;
+
+import java.io.Serializable;
+
+/**
+ * VariantContextVariantAdapter wraps the existing htsjdk VariantContext class so it can be
+ * used with the Variant API.
+ */
+public class VariantContextVariantAdapter implements Variant, Serializable {
+    private static final long serialVersionUID = 1L;
+
+    final private VariantContext variantContext;
+
+    public VariantContextVariantAdapter(VariantContext vc) { this.variantContext = vc; }
+    @Override
+    public String getContig() { return variantContext.getContig(); }
+    @Override
+    public int getStart() { return variantContext.getStart(); }
+    @Override
+    public int getEnd() { return variantContext.getEnd(); }
+    @Override
+    public boolean isSnp() { return variantContext.isSNP(); }
+    @Override
+    public boolean isIndel() { return variantContext.isIndel(); }
+    @Override
+    public String toString() {
+        return String.format("VariantContextVariantAdapter -- interval(%s:%d-%d), snp(%b), indel(%b)",
+                getContig(), getStart(), getEnd(), isSnp(), isIndel());
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        VariantContextVariantAdapter that = (VariantContextVariantAdapter) o;
+
+        return variantContext.equals(that.variantContext);
```

Comment:
Hey, remember that time I recommended using IntelliJ's auto-generated `equals()` and hashCode()` method? Yeah, not such good advice in this case, since `VariantContext` does not override `equals()`. Sorry about that.

Since (for now) we only need `equals()` for tests, I recommend that you:

1. Remove `equals()` and `hashCode()` from the two adapter classes.
2. Write a static utility method `variantsAreEqual(Variant v1, Variant v2)` in utils/variant/VariantUtils that checks equality only in terms of the existing interface methods.
3. Use this utility method in your `VariantContextVariantAdapterTest`